cvat/apps/engine/task.py

```diff
@@ -717,8 +717,8 @@ def _update_status(msg):
                         manifest.create()
                         _update_status('A manifest had been created')
 
-                        all_frames = len(manifest.reader)
-                        video_size = manifest.reader.resolution
+                        all_frames = len(manifest._reader)
+                        video_size = manifest._reader.resolution
```

Comment:
This seems odd; why did you remove the `reader` property if it was used here?

Reply:
Hmm, right. Thanks